Workbooks/Windows Virtual Desktop/AVD Insights/AVDWorkbookV2.workbook

```diff
@@ -1152,4 +1152,4 @@
     "Azure Monitor"
   ],
   "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/workbook.json"
-}
+}
```

Comment:
Revert this file if changes are not intentional

Reply:
Reverted the changes.